src/cljs/proton/layers/core/keybindings.cljs

```diff
@@ -142,9 +142,11 @@
                   :title "focus right pane"}
               :m {:action "maximize-panes:maximize"
                   :title "maximize pane"}
+              :n {:action "application:new-window"
+                  :title "new window"}
               :o {:title "close others"
                   :fx panes/close-other-panes}
-              :u {:action "open last-closed window"
```

Comment:
It appears that this command actually opens the most-recently closed tab (buffer) instead. I think it should be moved to  `SPC b u`.